src/components/Onboarding/Screens/HoneyswapLiquidity.js

```diff
@@ -1,19 +1,334 @@
+            `}
+          >
+            {denominatedAmount && (
+              <div
+                css={`
+                  font-weight: bold;
+                  display: flex;
+                  align-items: center;
+                `}
+              >
+                <span
+                  css={`
+                    margin-right: ${0.5 * GU}px;
+                  `}
+                >
+                  ≈
+                </span>
+                {hnyPriceLoading ? (
+                  <LoadingRing />
+                ) : (
+                  <span>
+                    {convertedValue}{' '}
+                    {denomination === HNY_DENOMINATION ? 'USD' : 'HNY'}
+                  </span>
+                )}
+              </div>
+            )}
+            <div
+              css={`
+                ${textStyle('body3')};
+              `}
+            >
+              You must specify an amount worth at least {MIN_HNY_USD} USD
+            </div>
+          </div>
+        </div>
+        <div
+          css={`
+            text-align: center;
+            ${textStyle('title2')};
+            margin: ${2 * GU}px 0;
+          `}
+        >
+          <span>+</span>
+        </div>
+        <div>
+          <Box padding={2 * GU}>
+            <div>Input</div>
+            <div
+              css={`
+                display: flex;
+                align-items: center;
+                justify-content: space-between;
+                margin-top: ${1.5 * GU}px;
+              `}
+            >
+              <TextInput
+                placeholder="0.0"
+                value={tokenAmount}
+                onChange={handleTokenAmountChange}
+                wide
+                css={`
+                  border: 0;
+                  width: 100%;
+                  padding: 0;
+                  ${textStyle('title3')};
+                  color: ${theme[tokenAmount ? 'content' : 'contentSecondary']};
+                `}
+              />
+              <img
+                src={getLocalTokenIconBySymbol(tokenSymbol)}
+                width="32"
+                height="32"
+                alt=""
+              />
+              <span
+                css={`
+                  ${textStyle('title4')};
+                  margin-left: ${1 * GU}px;
+                `}
+              >
+                {tokenSymbol}
+              </span>
+            </div>
+          </Box>
+          {liquidityProvided && (
+            <div
+              css={`
+                margin-top: ${3 * GU}px;
+                color: ${theme.contentSecondary};
+                ${textStyle('body3')};
+              `}
+            >
+              <div
+                css={`
+                  display: flex;
+                  align-items: center;
+                  justify-content: space-between;
+                `}
+              >
+                <span>Initial price</span>
+                <span>
+                  1 {tokenSymbol} ={' '}
```

Comment:
What do you think to show the USD value of 1 Garden token as well? Like `1GDN (10USD) = 0.50 HNY`

Reply:
Yes good idea 👍